code/game/objects/items/surgery_tray.dm

```diff
@@ -20,6 +20,7 @@
 		/obj/item/stack/sticky_tape/surgical,
 		/obj/item/surgical_drapes,
 		/obj/item/surgicaldrill,
+		/obj/item/autopsy_scanner,
```

Comment:
alphabetize your lists. there's an extension for VSCode that will automatically do this for you.